app/components/ArticlesDropdown/index.tsx

```diff
@@ -28,9 +28,17 @@ export const ArticlesDropdown = ({toc, categories}: ArticlesDropdownProps) => {
     </div>
   )
 
-  const ArticlesSection = ({category, toc}: {category: Category; toc: TOCItem[]}) => (
-    <div>
-      <div className="articles-dropdown-title">{category}</div>
+  const ArticlesSection = ({
+    category,
+    toc,
+    className,
+  }: {
+    category: Category
+    toc: TOCItem[]
+    className?: string
+  }) => (
+    <div className={'myClassName ' + (className || '')}>
```

Comment:
should this be `myClassName`?

Reply:
Right, no classes need to be applied to component at large so I removed 'myClassName ' + , hope that is the right move.